.github/workflows/node-ci-main.yml

```diff
@@ -0,0 +1,81 @@
+name: Node CI (landed on protected branch)
+
+on:
+  push:
+    branches:
+    - main
+    - changeset-release/*
+    - feature/*
+
+# Our jobs run like this to minimize wasting resource cycles:
+#   1. Prime caches for primary configuration (ubuntu on node 14).
```

Comment:
Maybe we don't even include the node version in this comment? Feels like it's easy to become dated. 
```suggestion
#   1. Prime caches for primary configuration (ubuntu on node 16).
```